Main.java

```diff
@@ -9,7 +11,50 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //Ask the user how many chargers they want
+    System.out.println("How many Chromebook Chargers are being purchased?");
+    //initialize how many chargers they are buying
+    int chromebookCharger = input.nextInt();
+
+    //ask the user how many motherboards they want
+    System.out.println("How many Motherboards are being purchased");
+    //initialize how many motherboards they are buying
+    int motherboard = input.nextInt();
+
+    //ask the user how many mice they want
+    System.out.println("How many Computer Mouses are being purchased");
+    //initialize how many mice they are buying
+    int computerMouse = input.nextInt();
+
+    //The cost of the items that are being purchased
+    final double chromebookChargerCost = 34.99;
+    final double motherboardCost = 127.50;
+    final double computerMouseCost = 18.00;
```

Comment:
Remember, constants are named using UPPER_CASE_LETTERING. but I do like the fact that you have used constants here!